.gitignore

```diff
@@ -0,0 +1,8 @@
+node_modules
+examples
+dist
+packages/ast/dist
+packages/backend/dist
+packages/frontend/dist
+packages/optimizer/dist
```

Comment:
I don't think these are necessary, adding `dist` should be enough. Please verify this. 

```suggestion
```